app/controllers/playlists_controller.rb

```diff
@@ -206,6 +207,13 @@ def update_multiple
     redirect_to edit_playlist_path(@playlist), notice: 'Playlist was successfully updated.'
   end
 
+  # PATCH/PUT /playlists/1
```

Comment:
It would be good to have this comment (and the one above) match the routes: 
# PATCH/PUT /playlists/1/regenerate_access_token